lib/call.ts

```diff
@@ -43,19 +62,28 @@ import { pause } from "./pause.ts";
  *
  * @param operator the operation, promise, async function, or generator funnction to call as part of this operation
  */
-export function call<T>(operator: () => Operation<T>): Operation<T>;
-export function call<T>(operator: () => Promise<T>): Operation<T>;
-export function call<T>(operator: Operation<T>): Operation<T>;
-export function call<T>(operator: Promise<T>): Operation<T>;
-export function call<T>(operator: Operator<T>): Operation<T> {
+export function call<T>(callable: Operation<T>): Operation<T>;
+export function call<T>(callable: () => Operation<T>): Operation<T>;
+export function call<T>(callable: () => Promise<T>): Operation<T>;
+export function call<T>(callable: () => T): Operation<T>;
+export function call<T>(callable: Callable<T>): Operation<T | Iterable<T>> {
```

Comment:
Why might this return an Operation<Iterable<T>>? If you have `call(() => [])` which override will it select? The one on line 68 or 69?

Reply:
You're right, removed.  Probably old code from when I was experimenting.